src/config/reader.rs

```diff
@@ -24,57 +22,15 @@ use crate::valid::{Valid, Validator};
 /// linked extensions to create a ConfigModule.
 pub struct ConfigReader {
     runtime: TargetRuntime,
```

Comment:
do we need to store runtime in config reader anymore?

Reply:
Hi @ssddOnTop,
thanks for the review.
https://github.com/tailcallhq/tailcall/blob/57ca2e423193899ac4582b5b6663844f151f02bc/src/config/reader.rs#L219
it being used here.
Any other way to get it?

Reply:
Let's keep it for now.